api/openapi.yaml

```diff
@@ -69,6 +69,13 @@ paths:
           application/json:
             schema:
               type: object
+              properties:
+                name:
```

Comment:
Suggest to make name and openapi required properties.

@topliceanurazvan Perhaps the wizard needs a third step to ask for these properties - we could autogenerate the name from the name of the API, but we have to make sure it doesn't collide with any existing names (which we should be able to do since the FE can retrieve the list of APIs). Namespace would be optional - default could be "default" ?

Reply:
Right now we have the API name and namespace on the first step which also includes the openapi spec.

![image](https://user-images.githubusercontent.com/47887589/161961315-fae47d8e-76d5-45a6-8bc6-1bea04802c87.png)

Good point with name collide, I guess here we should take into account the `namespace-name` combination as same names on different namespaces can be possible.
For the namespace we could have it required in the UI and have `default` as a default value and let the user change it if needed, even though I also like the idea of having the input optional.

Reply:
We could handle the namespace thing on the client and have it required in the openapi def, so that we always send a namespace in the body for the POST request?

Reply:
wdyt @jasmingacic @olensmar ?

Reply:
I think either is fine - whatever is easiest for you guys